### PR TITLE
Be able to run tests in development env w/o requiring a different dokerfile image

### PR DESCRIPTION
**Pull request by dgaubert**
cc/ @oleurud

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -1,142 +1,193 @@
...
+const fs = require('fs').promises;
 
 if (!process.env.NODE_ENV) {
     console.error('Please set "NODE_ENV" variable, e.g.: "NODE_ENV=test"');
     process.exit(1);
 }
-
 let configFileName = process.env.NODE_ENV;
-if (process.env.CARTO_SQL_API_ENV_BASED_CONF) {
+if (process.env.CARTO_WINDSHAFT_ENV_BASED_CONF) {
```
Comment (thedae) [inline test/index.js:12]: I think this env var is not correct, shouldn't it be `CARTO_SQL_API_ENV_BASED_CONF` ?

**Review by thedae — CHANGES_REQUESTED** [review]
Just a minor comment

File: test/index.js (hunk 1 above)
Comment (dgaubert) [reply test/index.js:12]: haha! `copy/paste`. Nice catch!

**Review by thedae — APPROVED** [review]
:+1:

**Pull request merged**